src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -1021,6 +1049,30 @@ public function iShouldBeAbleToGoToTheShippingStepAgain()
     }
 
     /**
+     * @Then I should be able to go to the complete step again
+     */
+    public function iShouldBeAbleToGoToTheCompleteStepAgain()
+    {
+        Assert::true(
+            $this->completePage->isOpen(),
```

Comment:
Is this proper check? If step is `I should be able to go to the complete step again` then I suppose we should check is there any button/link to that page or are there no errors after opening it. It's wrong assert or wrong step semantics.